Doc/library/urllib.parse.rst

```diff
@@ -451,13 +451,17 @@ task isn't already covered by the URL parsing functions above.
 .. function:: quote(string, safe='/', encoding=None, errors=None)
 
    Replace special characters in *string* using the ``%xx`` escape. Letters,
-   digits, and the characters ``'_.-'`` are never quoted. By default, this
+   digits, and the characters ``'_.-~'`` are never quoted. By default, this
    function is intended for quoting the path section of URL. The optional *safe*
    parameter specifies additional ASCII characters that should not be quoted
    --- its default value is ``'/'``.
 
    *string* may be either a :class:`str` or a :class:`bytes`.
 
+   .. versionchanged:: 3.7
+      Moving from RFC 2396 to RFC 3986 for quoting URL strings. Now, "~" is
```

Comment:
We typically write these notes in past tense, so a slight tweak here:

> Moved from RFC ...

And in the 2nd sentence:

> "~" is now included...